code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -609,13 +609,13 @@
 
 /datum/reagent/flightpotion/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message = 1)
 	if(iscarbon(M) && M.stat != DEAD)
-		if(!ishumanbasic(M) || reac_volume < 5) // implying xenohumans are holy
+		if(!(ishumanbasic(M) || islizard(M)) || reac_volume < 5) // implying xenohumans are holy
```

Comment:
Don't forget to remove this. Possibly even keep the holy trait for basic humans.